app/models/miq_schedule_worker/jobs.rb

```diff
@@ -120,7 +120,6 @@ def archived_entities_purge_timer
     queue_work(:class_name => "ContainerGroup", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerImage", :method_name => "purge_timer", :zone => nil)
     queue_work(:class_name => "ContainerProject", :method_name => "purge_timer", :zone => nil)
-    queue_work(:class_name => "ContainerDefinition", :method_name => "purge_timer", :zone => nil)
```

Comment:
can already queued ContainerDefinition purges cause problems?
(I have no idea what I'm talking about)

there are probably other scenarios where something is queued based on old schema, I don't know how even figure out if there is a problem :-)